jbpm-test-coverage/src/test/java/org/jbpm/test/functional/task/ProcessHumanTaskTest.java

```diff
@@ -83,6 +91,39 @@ public void testProcess() {
 		assertProcessInstanceNotActive(processInstance.getId(), ksession);
 	}
 	
+    @Test
+    public void testProcessWithHumanTaskAndCustomStrategy() {
+        createRuntimeManager("org/jbpm/test/functional/task/HumanTaskWithCustomAssignmentStrategy.bpmn2");
+        RuntimeEngine runtimeEngine = getRuntimeEngine();
+        KieSession ksession = runtimeEngine.getKieSession();
+        TaskService taskService = runtimeEngine.getTaskService();
+        AssignmentServiceRegistry.get().addStrategy(new AssignmentStrategy() {
+
+            @Override
+            public String getIdentifier() {
+                return "CustomStrategy";
+            }
+
+            @Override
+            public Assignment apply(Task task, TaskContext context, String excludedUser) {
+                return new AssignmentImpl("doctor");
+            }
+            
+        });
+        AssignmentServiceProvider.get().setEnabled(true);
```

Comment:
as I said, maybe this should be done through system properties instead?

Reply:
system property might affect other test.